runtime/indent/qml.vim

```diff
@@ -0,0 +1,59 @@
+" Vim indent file
+" Language:     QML
+" Maintainer:   Chase Knowlden <haroldknowlden@gmail.com>
+" Last Change:  2023 Aug 16
+"
+" Improved JavaScript indent script.
+
+" Indent script in place for this already?
+if exists("b:did_indent")
+  finish
+endif
+let b:did_indent = 1
+let b:undo_indent = "setlocal indentexpr< indkeys<"
```

Comment:
Should be `indentkeys<` not `indkeys<`.  Sorry for the typo in my earlier comment.